lib/typing/ext/time.go

```diff
@@ -20,7 +20,7 @@ func (e ExtendedTimeKindType) defaultLayout() (string, error) {
 	case TimestampTZKindType:
 		return time.RFC3339Nano, nil
 	case TimestampNTZKindType:
-		return RFC3339NanosecondNoTZ, nil
+		return RFC3339NoTZ, nil
```

Comment:
This is a deliberate change. The difference between the two are that `RFC3339NoTZ` uses `9` for formatting fractional seconds as opposed to `0`.

Reply:
The difference between `9` is that if there are no fractional seconds, instead of outputting fractional seconds with all zeroes, it'll simply omit it from the output.